next/next.config.js

```diff
@@ -1,10 +1,24 @@
 const { i18n } = require("./next-i18next.config");
+var crypto = require("crypto");
```

Comment:
Hei, Mario!
Sorry for uninvited comments, but Bernt posted link to this repo and out of curiosity I went to check PRs.
Why `var`?

Reply:
hi @aaleksu ! 

Good point, that can be a `const` . I have updated it in 39fa30bbcb1530f62e81e70f895c84ea50622a31

Thanks for noticing!